packages/ra-ui-materialui/src/button/DeleteWithConfirmButton.tsx

```diff
@@ -77,6 +84,15 @@ export const DeleteWithConfirmButton = <RecordType extends RaRecord = any>(
     if (isValidElement(recordRepresentation)) {
         recordRepresentation = `#${record?.id}`;
     }
+    const label =
```

Comment:
This is a regression: the label will be translated twice, because it's already translated by `Button`. Same for the other buttons.